packit.yaml

```diff
@@ -9,6 +9,18 @@ actions:
     - tools/webpack-jumpstart --wait --rebase
     - make XZ_OPT=-0 dist
     - sh -c 'echo cockpit-*.tar.xz'
+specfile_path: cockpit.spec
```

Comment:
Why is this necessary? As per the [docs](https://packit.dev/docs/configuration/), the implicit default should be downstream_package_name`.spec`, and `downstream_package_name` defaults to the github repo name ("cockpit").

Reply:
whoops, so it's a bug on our side then; it did not populate it correctly unless I explicitly specified the path

Reply:
I cannot reproduce locally :| so let's drop it and see

Reply:
reproduced, fixed; will rebase this PR once available in production